src/App.js

```diff
@@ -54,6 +57,181 @@ const App = () => {
           </div>
         </div>
       </section>
+      <section className="digits">
+        <h3 className="digits__title">Цифры и факты</h3>
+        <h2 className="digits__subtitle">Про учебу и мозг</h2>
+        <table className="digits__table">
+          <tr>
+            <td>
+              <p className="table__title">86 миллиардов</p>
+              <p className="table__text">Число нейронов в мозге человека</p>
+            </td>
+            <td>
+              <p className="table__title">2.1 миллиарда</p>
+              <p className="table__text">
+                Число нуждающихся в повышении квалификации
+              </p>
+              <p className="bank">Всемирный Банк, 2017</p>
+            </td>
+            <td>
+              <p className="table__title">1000 терабайт</p>
+              <p className="table__text">Объём памяти человека</p>
+            </td>
+            <td>
+              <p className="table__title">500 триллионов</p>
+              <p className="table__text">
+                Число ответственных за обучение нервных синапсов у взрослого
+                человека
+              </p>
+            </td>
+          </tr>
+          <tr>
+            <td>
+              <p className="table__title">420 миллионов</p>
+              <p className="table__text">
+                Число взрослых людей моложе 25 лет, не имеющих образования для
+                трудоустройства
+              </p>
+              <p className="bank">Всемирный Банк, 2017</p>
+            </td>
+            <td>
+              <p className="table__title">17–20 лет</p>
+              <p className="table__text">Пик обучаемости</p>
+            </td>
+            <td>
+              <p className="table__title">1885 год</p>
+              <p className="table__text">Открытие кривой забывания</p>
+            </td>
+            <td>
+              <p className="table__title">1889 год</p>
+              <p className="table__text">Открытие условного рефлекса</p>
+            </td>
+          </tr>
+        </table>
+      </section>
+      <section className="feynman" height="890px">
+        <h1 className="feynman__title">Метод Фейнмана</h1>
+        <h2 className="feynman__subtitle">Выучить и не забыть.</h2>
+        <img
+          className="feynman__portrait"
+          src={feynman}
+          width="867px"
+          height="637px"
+          alt="Portrait of Feynman"
+        />
+        <a href="https://en.wikipedia.org/wiki/Wiki" className="feynman__link">
+          Подробнее
+        </a>
+      </section>
+      <section className="kaufman">
+        <h3 className="kaufman__title">Принципы обучения</h3>
+        <h2 className="kaufman__subtitle">От Джоша Кауфмана</h2>
+        <table className="kaufman__table">
+          <tr>
+            <td>
+              <p className="kaufman__table__title">1</p>
```

Comment:
1. with BEM we name classes like `block__element` and element can be made of 1+ words like `block__element-title`
2. In feynman section you named classes like the following example, but in kaufman section the princin was changed. Both of them are good, but chose one. When I used BEM I prefered the kaufman section's style of naming.
```
feynman
-- feynman__table
---- table-title
```
...

Reply:
I think I need to use <h4>? I don't know if that what you mean, but I will do that

Reply:
< h4 >